worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/WorldEditPlugin.java

```diff
@@ -204,18 +133,23 @@ public void onLoad() {
             ChunkDeleter.runFromFile(delChunks, true);
         }
 
-        fail(() -> PermissionsResolverManager.initialize(INSTANCE), "Failed to initialize permissions resolver");
+        if(this.getDataFolder().getParentFile().listFiles(file -> {
+            if(file.getName().equals("DummyFawe.jar")){
+                return true;
+            }
+            return false;
+        }).length > 0){
+            getLogger().warning("DummyFawe detected! This is no longer necessary. Please remove immediately and restart your server!");
```

Comment:
We could just delete the file here notifying the jar has been deleted. The chance people check their logs looking for issues is pretty low.
If DummyFawe is present, it will trigger `Ambiguous plugin name...` anyway since the name is the same.